src/CMakeLists.txt

```diff
@@ -506,7 +506,7 @@ if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
       find_package(pmem REQUIRED COMPONENTS pmem)
     endif()
     if(WITH_RBD_RWL)
-      find_package(pmem REQUIRED COMPONENTS pmemobj)
+      find_package(pmemobj REQUIRED COMPONENTS pmemobj)
```

Comment:
Does this change the references to `pmem::pmemobj` (i.e. would they now need to be `pmemobj::pmemobj`)?